packages/core-accountsjs/src/module/configureAccountsModule.ts

```diff
@@ -100,15 +101,33 @@ export const configureAccountsModule = async ({
       };
     },
 
-    createImpersonationToken: async (userId, rawContext) => {
+    createImpersonationToken: async (userId, rawContext: Context) => {
       // TODO: rawContext does not contain user and locale date anymore
       // following the type but the code still depends on it
-      const context = evaluateContext(filterContext(rawContext));
+      const { modules } = rawContext;
```

Comment:
Why not use the evaluateContext(filterContext thing? If it's not needed you could remove it everywhere it's used currently to clean it up.

Reply:
`modules` will be filtered after `evaluateContext(filterContext` operation but we actually only need access to modules, not a normalized context and we cant remove the dependency of this utility functions because they are used to normalize the final token information stored in DB